style.css

```diff
@@ -1,514 +1,1134 @@
+    color: #172b4d;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .counter {
+    padding: 0;
+    display: flex;
+    align-items: center;
+    gap: 12px;
+    left: 24px;
+    top: 24px;
+    border-radius: 0.01px;
+  }
+
+  .canopy {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.01em;
+    text-align: left;
+    color: #344563;
+  }
+
+  .canopy-text {
+    display: flex;
+    padding: 0;
+    justify-content: left;
+    gap: 1.25rem;
+    letter-spacing: 0.05em;
+    list-style-type: none;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    color: #6b778c;
+  }
+
+  .primary-text {
+    font-weight: 400;
+    line-height: 24px;
+    letter-spacing: 0.01em;
+    color: #344563;
+    font-size: 16px;
+    text-align: left;
+  }
+
+  .programming-languages {
+    color: solid #000;
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+    gap: 12px;
+    padding: 0;
+  }
+
+  .languages .programming-languages .btn:hover {
+    color: #75a20c;
+  }
+
+  .languages a {
+    border-radius: 8px;
+    padding: 4px 12px;
+    background-color: #ebebff;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 3%;
+    text-align: left;
+    color: #6070ff;
+    text-decoration: none;
+  }
+
+  .text-see-project {
+    margin: 0.75rem 0;
+    padding-top: 12px;
+  }
+
+  .text-see-project a {
+    text-decoration: none;
+    padding-top: 12px;
+    font-size: 17px;
+    font-weight: 500;
+  }
+
+  .see-project-button {
+    border-radius: 8px;
+    padding: 12px;
+    border: solid 1px #396df2;
+    gap: 10px;
+    top: 12px;
+    color: #396df2;
+    line-height: 24px;
+    letter-spacing: 3%;
+    text-align: center;
+  }
+
+  .text-see-project-desktop {
+    width: 19%;
+    padding: 12px;
+    border-radius: 8px;
+    background-color: #4053fc;
+  }
+
+  .see-project-btn {
+    text-decoration: none;
+    font-size: 17px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    text-align: center;
+    color: #fff;
+  }
+
+  .text-see-project-desktop:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .hand-cursor img {
+    padding-left: 320px;
+    padding-right: 85px;
+  }
+
+  .see-project-button :active {
+    color: #fff;
+  }
+
+  .text-see-project .see-project-button:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .text-see-project .see-project-button:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .header-bar button {
+    background-color: transparent;
+    border: none;
+  }
+
+  .about-myself-section {
+    height: 750px;
+    display: flex;
+    align-items: center;
+    color: #344563;
+    border-radius: 16px;
+    padding: 16px;
+    margin: auto;
+    gap: 12px;
+    background-color: #fff;
+    border-top-right-radius: 90px;
+    border-bottom-left-radius: 0;
+    margin-bottom: 0;
+  }
```

Comment:
- [ ] This section should be justified center which will give space to the left and right just as it is in the figma design.
- [ ] Also, the content should be 40% width for the left and 60% width for the right but yours is vis-a-vis. Kindly correct that too.
### Your work
![image](https://user-images.githubusercontent.com/86200356/207843284-ff83c9b1-4ec1-4057-a2d2-d66de8ff635a.png)

### Figma

![image](https://user-images.githubusercontent.com/86200356/207843603-9c5d40f2-1176-412f-bca3-ee115826c659.png)

Reply:
@Olamarx Thanks for the review! I have tried to use 60% width for the right and it worked but 40% for the left does not create the required space on the left side of the content.

Reply:
@Olamarx I have made the changes. Please check it.